README.md

```diff
@@ -133,6 +133,12 @@ For Fedora 29+, `sudo dnf install nmstate`.
 For others distribution, please see the [install](README.install.md)
 instructions.
 
+## Documentation
+
+It is possible to consult the libnmstate python API
```

Comment:
How about this:

- [libnmstate API](https://nmstate.github.io/devel/api.html)
- [Code examples](https://nmstate.github.io/devel/py_example.html)
- [State examples](https://nmstate.github.io/examples.html)
- [nmstatectl user guide](https://nmstate.github.io/cli_guide.html)
- nmstatectl man page: `man nmstatectl`

Reply:
Done! Thanks